### PR TITLE
Migrate example pages to documentation

### PR DESCRIPTION
**Pull request by Sergiodero**
This update moves all examples (Schedule & Realtime) previously hosted in the Resource Library to the Technical Documentation section. It also updates the navigation to reflect these changes.

These changes might be bundled up with the Overview update from PR #16 

Do you might having a look at it @isabelle-dr ?

### CODE REVIEW

**Review by isabelle-dr — COMMENTED** [review]
Thank you for having these in alphabetical order! It always bothered me on the current gtfs.org. 

I think we're missing the [Routes, stops, and trips](https://gtfs.org/schedule/examples/routes-stops-trips/) example, no?
Also, are we hosting the [Example GTFS feed](https://gtfs.org/schedule/example-feed/) somewhere else?

The changes to overview.md are indeed included there, but they represent the latest no it shouldn't cause problems.

**Comment by Sergiodero** [comment]
Thanks @isabelle-dr I've just added the Routes, Stops and Trips example in the latest commit. As for the [Example GTFS feed](https://gtfs.org/schedule/example-feed/), this one is currently hosted at the end of the getting started section, I think it makes sense to leave it there as a sample, any thoughts? 

@tzujenchanmbd do you mind taking a look as well? thank you both!

File: docs/documentation/schedule/examples/routes-stops-trips.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+```
+route_id,service_id,trip_id,trip_headsign,direction_id,shape_id
+303-20670,weekend_service,60270564,"MAX ORANGE SADDLETOWNE",0,3030026
+303-20670,weekend_service,60270565,"MAX ORANGE BRENTWOOD",1,3030027
+303-20670,weekend_service,60270566,"MAX ORANGE BRENTWOOD",1,3030027
+```
+
+- The `route_id` from [routes.txt](../../reference/#routestxt) that corresponds to MAX Orange is listed
+- The `service_id` from [calendar.txt](../../reference/#calendartxt) that corresponds to weekends is listed
+- Each records contains a unique ID for each trip
```
Comment (tzujenchanmbd) [inline docs/documentation/schedule/examples/routes-stops-trips.md]:
```suggestion
- Each records contains a unique ID for each trip.
```

File: docs/documentation/schedule/examples/routes-stops-trips.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+The headsign text is provided, which is what is typically displayed on signs inside and outside the the bus
+- The field `direction_id` allows for a distinction between trips of the same route going in different directions. For example, distinguishing between inbound trips and outbound trips - or southbound trips and northbound trips. 
+    - In this case, trips towards Saddletowne have a `direction_id=0` and trips towards Brentwood have a `direction_id=1`. The values in direction_id have no inherent meaning, they are only used to assign one direction of travel versus another
+- The `shape_id` from [shapes.txt](../../reference/#shapestxt) that corresponds to the MAX Orange route towards Saddletowne is listed for the first record and the one for MAX Orange route towards Brentwood is listed for the second and third records
+
+
+The `shape_id=3030026` corresponds to the MAX Orange towards Saddletowne. The file below includes information on the points that outline the route as well as the distance between those points. With this information, it is possible to plot the route on a map for trip planning or analytics purposes.
```
Comment (tzujenchanmbd) [inline docs/documentation/schedule/examples/routes-stops-trips.md]: > The file below includes information on the points that outline the route as well as the distance between those points.

- points that outline the "trip"
- "distance between those point" may confuse ppl? (`shape_dist_traveled` is for the distance from the first shape point to the point specified in this record)

File: docs/documentation/schedule/examples/routes-stops-trips.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+| `weekend_service1` | `20220623` | `20220716` |
+| `weekend_service2` | `20220718` | `20220903` |
+
+However, this complicates the file since `service_id` is broken in two and this break will cascade to [trips.txt](../../reference/#tripstxt). Instead, this can be done in an easier way using [calendar_dates.txt](../../reference/#calendar_datestxt) as shown below:
+
+[**calendar_dates.txt**](../../reference/#calendar_datestxt) 
+
+```
+service_id,date,exception_type
+weekend_service,20220623,2
```
Comment (tzujenchanmbd) [inline docs/documentation/schedule/examples/routes-stops-trips.md]:
```suggestion
weekend_service,20220717,2
```

**Review by tzujenchanmbd — COMMENTED** [review]
Please see the following suggestions, thanks!

File: docs/documentation/schedule/examples/routes-stops-trips.md (hunk 2 above)
Comment (Sergiodero) [reply docs/documentation/schedule/examples/routes-stops-trips.md]: Updated the text to: "The file below includes information on the points that outline the trip as well as the distance between each point and the start of the trip."

**Comment by Sergiodero** [comment]
Thanks @tzujenchanmbd! changes have been made to address your comments. Can any of you have a final look before approving the changes? Thank you both!

**Review by tzujenchanmbd — APPROVED** [review]
lgtm!

**Review by isabelle-dr — APPROVED** [review]

**Pull request merged**